pkg/storaged/format-dialog.jsx

```diff
@@ -246,9 +246,6 @@ export function format_dialog(client, path, start, size, enable_dos_extended) {
     add_fsys("ext4", { value: "luks+ext4", title: _("Encrypted EXT4 (LUKS)") });
     add_fsys("vfat", { value: "vfat", title: "VFAT" });
     add_fsys("ntfs", { value: "ntfs", title: "NTFS" });
-    add_fsys(true, { value: "dos-extended",
-                     title: _("Extended Partition"),
-                     disabled: !(create_partition && enable_dos_extended) });
```

Comment:
You can create logical partitions: 
![image](https://user-images.githubusercontent.com/3228183/57678128-39666e00-7631-11e9-9848-16da4cf448ca.png)

I think we should keep support for displaying MBR partition tables, but we don't need to support creating them.  Which means we should also remove the choice from the "Create Partition Table" menu.  Maybe do this all in a separate PR?

Reply:
Compromise: Hide the menu entry entirely instead of just disabling it.  This gives the same UI cleanup for almost everyone without actually removing functionality.